ios/Hubbub/LoginViewController.swift

```diff
@@ -9,6 +9,8 @@
 import Firebase
 import p2_OAuth2
 import UIKit
+import Crashlytics
```

Comment:
Good point. Let's delete the crash button, but lines 81-83 add a "log in" Event to Answers on Fabric. If we like that, then we'd keep it for Answers.logLogin... What do you think?